test/e2e/dra/test-driver/app/gomega.go

```diff
@@ -33,6 +33,15 @@ var BeRegistered = gcustom.MakeMatcher(func(actualCalls []GRPCCall) (bool, error
 	return false, nil
 }).WithMessage("contain successful NotifyRegistrationStatus call")
 
+var GetInfoFailed = gcustom.MakeMatcher(func(actualCalls []GRPCCall) (bool, error) {
+	for _, call := range actualCalls {
+		if call.FullMethod == "/pluginregistration.Registration/GetInfo" && call.Err != nil {
+			return true, nil
+		}
+	}
+	return false, nil
+}).WithMessage("contain unsuccessful GetInfo call")
+
```

Comment:
Can we use this opportunity to move the gomega matchers into a `test/e2e/dra/test-driver/testing` package which then gets imported as `testdrivertesting`?

They don't belong here and the usage as `testdriver.GetInfoFailed` looked confusing until I realised that this is a matcher, not some function.

Alternatively, `gomega` instead of `testing` also works. It would avoid the `test` + `testing` repetition.

Side note: it might be better to turn `GetInfoFailed` into a function which returns a new instance. Some Gomega matchers store information, which can become confusing when the same instance gets used multiple times. I don't know whether that is a problem here, but let's be on the safe side.

Reply:
Can I do it in a separate PR?

Reply:
After this one is merged :)

Reply:
Why not now? Then we don't need to bother approvers twice.

Reply:
> Why not now?

Because it's out of scope of this PR?

Reply:
Adding `GetInfoFailed` here is not good, so doing something better is in scope for this PR.

If you feel that a separate cleanup PR is necessary, then do that first and we wait with this PR until the cleanup PR is merged.

Reply:
But I think a single PR with several commits would be less work for everyone involved. Up to you.

Reply:
Thank you for the review!
I've updated this PR as you've suggested. PTAL